kernelci/config/lab.py

```diff
@@ -84,7 +90,25 @@ def queue_timeout(self):
     def from_yaml(cls, lab, kw):
         priority = lab.get('priority')
         if priority:
-            kw['priority'] = priority
+            if priority == 'low':
+                priority = 0
+            elif priority == 'medium':
+                priority = 50
+            elif priority == 'high':
+                priority = 100
+
+            # If min/max are specified these will be overridden
+            kw['priority_min'] = priority
+            kw['priority_max'] = priority
+
+        # 0 is a valid value so explicitly check for None
+        priority_min = lab.get('priority_min')
+        if priority_min is not None:
+            kw['priority_min'] = priority_min
```

Comment:
I haven't tested this but it looks like the YAML values are always parsed as strings even if they're only digits, in which case they would need to be converted to integer here:
```suggestion
            kw['priority_min'] = int(priority_min)
```